frontend/lib/dataService.tsx

```diff
@@ -0,0 +1,12 @@
+export async function getCities(id:string = "") {
+    const slug = id ? `/${id}` : ""
+    const cities = await (await fetch(`${process.env.REST_API}/api/cities${slug}`, {
+        // TODO: proper url
```

Comment:
This TODO has moved here from somewhere else. Do we have a plan for how to handle it..?

Reply:
The Todo was to add a env variable for the rest API, which was some forgot to remove the Todo. Will remove it.